biz.aQute.bndlib/src/aQute/bnd/build/BndProgress.java

```diff
@@ -0,0 +1,36 @@
+package aQute.bnd.build;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import aQute.bnd.service.progress.ProgressPlugin;
+
+public class BndProgress implements ProgressPlugin {
+
+	private final Logger logger = LoggerFactory.getLogger(getClass().getName());
+
+	@Override
+	public Task startTask(final String name, final int size) {
+		if (logger.isWarnEnabled()) { // warn is disabled in gradle quiet mode
+			System.out.println(name);
```

Comment:
Since you have a logger, why not log to that logger instead of System.out. Gradle will pick this up just fine as it does for logging from the Processor.